packages/react-dom/src/events/SyntheticClipboardEvent.js

```diff
@@ -13,9 +13,11 @@ import SyntheticEvent from 'events/SyntheticEvent';
  */
 var ClipboardEventInterface = {
   clipboardData: function(event) {
-    return 'clipboardData' in event
-      ? event.clipboardData
-      : window.clipboardData;
+    if ('clipboardData' in event) {
+      return event.clipboardData;
+    }
+    var doc = (event.target && event.target.ownerDocument) || document;
+    return doc.defaultView.clipboardData;
   },
 };
```

Comment:
What does clipboard data have to do with selection?

Reply:
Nothing to do with selection; this is just another part of the codebase that relies on the global `window` object rather than the `window` object relative to the DOM node being used. That said, I just looked it up, and `window.clipboardData` is an IE-only API and one where it seems like the code being executed should only try to look up it’s own `window.clipboardData` (because of browser permissions / security settings), so I’m going to revert this set of changes.

Reply:
Ah still, do you think it's worth landing? What do you think about sending this out in a separate PR?

Reply:
I don’t know really know anything about clipboard APIs in IE, but based on a quick google search, my hunch is that this change wouldn’t actually be for the better. There’s some mentions about browser settings and issues with clipboardData not being accessible across different browsing contexts, so keeping it hardcoded to `window.clipboardData` might actually be correct.

But I’d be happy to open a separate PR with it if someone would be able to figure that all out.